src/content/docs/pt-br/guides/upgrade-to/v4.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Atualize para o Astro v4
+description: Como atualizar seu projeto para a versão mais recente do Astro (v4.0).
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+Este guia vai te ajudar a migrar do Astro v3 para o Astro v4.
+
+Precisa atualizar um projeto antigo para a v3? Veja nosso [antigo guia de migração](/pt-br/guides/upgrade-to/v3/).
+
+Precisa ver a documentação da v3? Visite essa [versão antiga do site de documentação (snapshot v3.6 não mantido)](https://docs-git-v3-docs-unmaintained-astrodotbuild.vercel.app/).
+
+## Atualize o Astro
+
+Atualize a versão do Astro e de todas as integrações oficiais em seu projeto para a versão mais recente utilizando seu gerenciador de pacotes.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Atualiza o Astro e integrações oficiais juntos
+  npx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Atualiza o Astro e integrações oficiais juntos
+  pnpm dlx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Atualiza o Astro e integrações oficiais juntos
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Você também pode [atualizar suas integrações do Astro manualmente](/pt-br/guides/integrations-guide/#upgrade-manual) se preferir, e você pode precisar atualizar outras dependências no seu projeto.
```

Comment:
Would the link work if I point to the pt-br page with the English heading since that is what is shown as a fallback?

When someone translates that page it would immediately flag that this link needs to be updated

Reply:
I wouldn't like to block this PR waiting for a translation of the integrations guide.

Leaving without the link would make it hard to remember this and have the pages in sync, it is the sort of thing that could be automated.

I'll think about something to help with that in the future, for now I'll leave the link to the English page

Reply:
It seems like it is impossible to have this link:
- If I link to the Portuguese page, the check breaks with "link not using canonical url" since it is acting as an alias to the English page
- If I link to the English page, the check breaks with "link to unexpected language"

So I'll leave without the link for now. If there is any other alternative I'm happy to change it